pkg/kstatus/polling/statusreaders/testing.go

```diff
@@ -12,8 +12,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/cli-utils/pkg/kstatus/polling/engine"
 	"sigs.k8s.io/cli-utils/pkg/kstatus/polling/event"
-	"sigs.k8s.io/cli-utils/pkg/kstatus/polling/testutil"
 	"sigs.k8s.io/cli-utils/pkg/object"
+	"sigs.k8s.io/cli-utils/pkg/testutil"
```

Comment:
Please make this unrelated change in another PR.

Reply:
this should be outdated... the commit is gone. not sure what happened to github that it thinks this is current...

Reply:
fixed.